lib/uffizzi/cli/connect.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'uffizzi'
+
+module Uffizzi
+  class CLI::Connect
+    include ApiClient
+
+    def run(credential_type, credential_file_path)
+      case credential_type
+      when 'docker-hub'
+        handle_docker_hub
+      when 'acr'
+        handle_azure
+      when 'ecr'
+        handle_amazon
+      when 'gcr'
+        handle_google(credential_file_path)
+      else
+        Uffizzi.ui.say('Unsupported credential type.')
+      end
+    end
+
+    private
+
+    def handle_docker_hub
+      IO::console.write('Username: ')
+      username = IO::console.gets.strip
+
+      password = IO::console.getpass('Password: ')
+
+      params = {
+        username: username,
+        password: password,
+        type: Uffizzi.configuration.credential_types[:dockerhub],
+      }
+
+      hostname = ConfigFile.read_option(:hostname)
+      response = create_credential(hostname, params)
+
+      if ResponseHelper.created?(response)
+        Uffizzi.ui.say('Successfully connected to DockerHub')
+      else
+        ResponseHelper.handle_failed_response(response)
+      end
+    end
+
+    def handle_azure
+      IO::console.write('Registry Domain: ')
+      registry_url = prepare_registry_url(IO::console.gets.strip)
+
+      IO::console.write('Docker ID: ')
+      username = IO::console.gets.strip
+
+      password = IO::console.getpass('Password/Access Token: ')
+
+      params = {
+        username: username,
+        password: password,
+        registry_url: registry_url,
+        type: Uffizzi.configuration.credential_types[:azure],
+      }
+
+      hostname = ConfigFile.read_option(:hostname)
+      response = create_credential(hostname, params)
+
+      if ResponseHelper.created?(response)
+        Uffizzi.ui.say('Successfully connected to ACR')
```

Comment:
Do you think it makes sense to create a method like:
`print_success_message(connection_name)`